workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowRunsTabContent.tsx

```diff
@@ -57,6 +61,8 @@ const makeSelectItemsFromProcessInstanceValues = () =>
   );
 
 export const WorkflowRunsTabContent = () => {
+  const { workflowId } = useRouteRefParams(workflowRouteRef);
+  const statuses = makeSelectItemsFromProcessInstanceValues();
```

Comment:
with this change, the statuses are recalculated for every render...
what @mareklibra meant was to move this to be a constant outside the component, right under the function definition of makeSelectItemsFromProcessInstanceValues